specification/sections/05-events.md

```diff
@@ -197,4 +214,22 @@ see: [provider event types](../types.md#provider-events), [provider events](#51-
 
 The `PROVIDER_ERROR` is not emitted from the provider itself; the SDK implementation must run the `PROVIDER_ERROR` handlers if the `on context changed` throws or otherwise signals an error.
 
-see: [provider event types](../types.md#provider-events), [provider events](#51-provider-events), context, [provider context reconciliation](02-providers.md#26-provider-context-reconciliation)
\ No newline at end of file
+see: [provider event types](../types.md#provider-events), [provider events](#51-provider-events), context, [provider context reconciliation](02-providers.md#26-provider-context-reconciliation)
```

Comment:
Should `context` be a link?

Reply:
I think this is actually a typo, I think just linking to `provider context reconciliation` is enough.